VideoLocker/res/layout/row_course_list.xml

```diff
@@ -220,16 +175,35 @@
         </LinearLayout>
     </RelativeLayout>
 
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/course_certified_banner"
+        style="@style/bold_white_text"
+        android:layout_width="wrap_content"
```

Comment:
hmm ... I think that `RelativeLayout` wasn't meant to be removed. Following are the `before` and `after` looks of the screen without/with the change:

BEFORE:
![screenshot from 2015-09-11 12 57 46](https://cloud.githubusercontent.com/assets/1710804/9809819/00ecce3e-5885-11e5-88f9-83ee929f5477.png)

AFTER:
![screenshot from 2015-09-11 12 56 03](https://cloud.githubusercontent.com/assets/1710804/9809821/0807a32e-5885-11e5-9a85-c342a56b3e75.png)

I think I should revert it back to `BEFORE`, i guess ?